src/router_core/connections.c

```diff
@@ -1443,6 +1443,7 @@ qdr_link_t *qdr_connection_new_streaming_link_CT(qdr_core_t *core, qdr_connectio
 
     if (out_link) {
         out_link->streaming = true;
+        out_link->priority = 4;
```

Comment:
Should we instead default the priority field in the qdr_link to 4  ?

Reply:
We could - this was the least-risk minimal fix in case we needed to backport to 1.15.0.

Going forward we probably need to consider making all router control traffic a higher priority than any user-generated traffic.